build/webpack.base.conf.js

```diff
@@ -41,7 +41,9 @@ module.exports = {
 			'@': resolve('src'),
 			'foundation': 'foundation-sites/js',
 			// alias promise module to handle timesync calling require('promise')
-			'promise': resolve('build/promise.js')
+			'promise': resolve('build/promise.js'),
+			// required for src/components/Kv/KvRichTextRenderer.vue
+			'vue$': 'vue/dist/vue.esm.js'
```

Comment:
The entire vue compiler is required here. Bundle comparison shows 33.27 kb for the full compiler, and 22.79 kb for our previous runtime only Vue.